DEVELOPING.md

```diff
@@ -17,7 +32,131 @@ This project:
 
 3. uses [Swift Package Manager](https://github.com/apple/swift-package-manager) to manage dependencies for development. Cocapods is supported for *consuming* the SDK.
 
-4. uses [secret-shield](https://github.com/mapbox/secret-shield) to help block secrets such as access tokens from being exposed. Install `secret-shield` by entering `npm install -g @mapbox/secret-shield`. Install the pre-commit hook by running `scripts/install-pre-commit/install-pre-commit.sh`
+4. uses [secret-shield](https://github.com/mapbox/secret-shield) to help block secrets such as access tokens from being exposed. Setup `secret-shield` by running:
+```
+npm install -g @mapbox/secret-shield
+scripts/install-pre-commit/install-pre-commit.sh
+```
+
+5. uses CircleCI and AWS Device Farm for continuous integration.
+
+### Accessing the Maps SDK's source
+
+Clone the git repository:
+
+```
+git clone git@github.com:mapbox/mapbox-maps-ios.git && cd mapbox-maps-ios
+```
+
+### Building the Maps SDK
+
+Pre-requisite: Valid `.netrc` file located on your machine at `~/.netrc`. Please visit https://docs.mapbox.com/ios/maps/overview for additional information about setting up your `.netrc` file.
```

Comment:
Let's explain why this is necessary: it allows Swift Package Manager to download binary dependencies from Mapbox SDK Registry.